src/components/SidebarNavItem/SidebarNavItem.js

```diff
@@ -24,7 +24,7 @@ class SidebarNavItem extends PureComponent {
 
   onGetActive(name, pathname) {
     const { theme } = this.props;
-    return pathname.includes(name) ? theme.sidebar.itemActive : '';
+    return pathname.slice(1) === name ? theme.sidebar.itemActive : '';
```

Comment:
Is `pathname` a list?

Reply:
It's a string and the first character is `/`. It comes from the browser location and gets passed through via React Router